content/docs/command-reference/checkout.md

```diff
@@ -33,15 +38,16 @@ for more details.
 
 The execution of `dvc checkout` does the following:
 
-- Scans the `.dvc` and `dvc.lock` files to compare against the data files or
-  directories in the <abbr>workspace</abbr>. DVC knows which data
-  (<abbr>outputs</abbr>) match because the corresponding hash values are saved
-  in the `outs` fields in those files. Scanning is limited to the given
-  `targets` (if any). See also options `--with-deps` and `--recursive` below.
+- Uses the appropriate `dvc.lock` and `.dvc` files to compare the hash values of
```

Comment:
what is the appropriate `dvc.lock`?

I think like with an entry paragraph, it was not perfect but more explicit and easier to understand.

Reply:
"appropriate" is no longer being used here, it has been simplified.